Cargo.toml

```diff
@@ -33,8 +33,12 @@ trybuild = "1.0.23"
 rustversion = "1.0"
 
 [features]
-default = ["macros"]
+default = ["macros", "all-apis"]
 macros = ["ctor", "indoc", "inventory", "paste", "pyo3cls", "unindent"]
+# Enable the use of `not(Py_LIMITED_API)` s.
+# This is incompatible with abi3(PEP384). See https://www.python.org/dev/peps/pep-0384/ more.
+all-apis = []
```

Comment:
I think it would be better to stick to the (c)python terminology of "stable abi"/"limited api"/"abi3" or an inverted term such as "unstable abi" or "unlimited api".

Reply:
Thanks, I choose `unstable-api`